x/slashing/autocli.go

```diff
@@ -36,5 +36,19 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 				},
 			},
 		},
+		Tx: &autocliv1.ServiceCommandDescriptor{
+			Service: slashingv1beta.Msg_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Unjail",
```

Comment:
I need to add capability to use another address codec when setting the from address. Here's it is expected to be a validator address